locales/en/allocations.json

```diff
@@ -1,7 +1,6 @@
 {
-  "complete": "complete",
-  "incomplete": "incomplete",
-  "total": "total allocations",
+  "total": "total allocation",
```

Comment:
"1 total allocation" sounds quite odd. But what else could it be?

Reply:
Yep, doesn't read quite as well. But equally "1 total allocations" feels incorrect. 

I think Andy may have toyed with just "total" but that's not quite as descriptive 🤷‍♂️